airbyte-ci/connectors/pipelines/pipelines/bases.py

```diff
@@ -620,3 +630,6 @@ def print(self):
 
         main_panel = Panel(Group(*to_render), title=main_panel_title, subtitle=duration_subtitle)
         console.print(main_panel)
+
+        if self.pipeline_context.dagger_cloud_url:
+            self.pipeline_context.logger.info(f"🔗 View commit in Dagger Cloud: {self.pipeline_context.dagger_cloud_url}")
```

Comment:
Ideally i wanted to add this as a section in the Panel printed with console.print, but for the life of me I couldn't figure out how to make the link not get cut off. In CI it kept adding a newline which made the link not clickable.